canonical-web-and-design/canonical.com
# Add redirect

## Done

This link is in the footer of launchpad the career one https://launchpad.net/

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/about-canonical/careers
- should go to /careers/all
